harvestingkit/elsevier_package.py

```diff
@@ -295,7 +295,7 @@ def _normalize_article_dir_with_dtd(self, path):
     def _add_references(self, xml_doc, rec, refextract_callback=None):
         for label, authors, doi, issue, page, title, volume, year,\
```

Comment:
I did not write the code, I'm just trying to fix a bug. I am not planning to rewrite the whole program with HEPCrawl arround the corner.